pkgs/applications/office/paperless/default.nix

```diff
@@ -0,0 +1,160 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, callPackage
+
+, python36
+, file
+, imagemagickBig # includes ghostscript
+, optipng
+, poppler
+, tesseract
+, unpaper
+}:
+
+## Usage
+
+# ${paperless}/bin/paperless wraps manage.py
+
+# ${paperless}/share/paperless/setup-env.sh can be sourced from a
+# shell script to setup a Paperless environment
+
+# paperless.withConfig is a convenience function to setup a
+# configured Paperless instance. (See ./withConfig.nix)
+
+# For WSGI with gunicorn, use a shell script like this:
+#   source ${paperless}/share/paperless/setup-env.sh
+#   PYTHONPATH=$paperlessSrc gunicorn paperless.wsgi
+
+let
+  paperless = stdenv.mkDerivation rec {
+    name = "paperless-${version}";
+    version = "2.6.0";
+
+    src = fetchFromGitHub {
+      owner = "danielquinn";
+      repo = "paperless";
+      rev = version;
+      sha256 = "160idy0vwzkb134axkiyj3yf85cgwggz0846w99fb7nckhx7v51m";
+    };
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    phases = [ "unpackPhase" "patchPhase" "buildPhase" "checkPhase" ];
```

Comment:
I don't think we want to be doing this anymore https://github.com/NixOS/nixpkgs/issues/28910

Reply:
Fixed: https://github.com/NixOS/nixpkgs/commit/0dd454109793cea5226c2d0bfd2f9cbd587273e6